apis/python/src/tiledbsoma/indexed_dataframe.py

```diff
@@ -200,18 +199,57 @@ def read(
                 query_condition=query_condition,
             )
 
+            # Acceptable ways to index:
```

Comment:
these comments belong in the function docstring, so the end user benefits (and we keep them up-to-date :-)).